### ISSUE TITLE
Add 'elasticsearch_extra_options' var for extra options in config

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-elasticsearch#38]
For a simple way to configure Elasticsearch options which are not exposed in this rather simple role, I would like to add an `elasticsearch_extra_options` variable, which is basically dropped into the end of the `elasticsearch.yml` template file.

Users could then add options to the file like:

```yaml
elasticsearch_extra_options: |
  some.option: true
  another.option: false
```

**Comment by jamieburchell** [comment]
It would be very useful to have https://github.com/geerlingguy/ansible-role-elasticsearch/pull/41 merged so we can add additional configuration.

**Comment by dale-c-anderson** [comment]
@geerlingguy , see https://github.com/geerlingguy/ansible-role-elasticsearch/pull/54 , which is exactly what you suggested. It worked like a charm for my dedicated 7.x elasticache node in a small network.

**Comment by geerlingguy** [comment]
Fixed via #54, thanks!
